components/CodeBlock.tsx

```diff
@@ -1,24 +1,43 @@
-/* Copyright 2020 the Deno authors. All rights reserved. MIT license. */
+// Copyright 2022 the Deno authors. All rights reserved. MIT license.
 
-import React, { useEffect, useState } from "react";
-import Highlight, { Prism } from "prism-react-renderer";
-import light from "prism-react-renderer/themes/github";
-import { useLayoutEffect } from "react";
+/** @jsx h */
+/** @jsxFrag Fragment */
+import { Fragment, h, htmlEscape, Prism } from "../deps.ts";
+import { normalizeTokens } from "../util/prism_utils.ts";
+
+import "https://esm.sh/prismjs@1.25.0/components/prism-bash?no-check";
+//import "https://esm.sh/prismjs@1.25.0/components/prism-batch?no-check";
+import "https://esm.sh/prismjs@1.25.0/components/prism-css?no-check";
+//import "https://esm.sh/prismjs@1.25.0/components/prism-css-extras?no-check";
+//import "https://esm.sh/prismjs@1.25.0/components/prism-editorconfig?no-check";
+import "https://esm.sh/prismjs@1.25.0/components/prism-diff?no-check";
+//import "https://esm.sh/prismjs@1.25.0/components/prism-docker?no-check";
+//import "https://esm.sh/prismjs@1.25.0/components/prism-git?no-check";
+//import "https://esm.sh/prismjs@1.25.0/components/prism-ignore?no-check";
+import "https://esm.sh/prismjs@1.25.0/components/prism-javascript?no-check";
+//import "https://esm.sh/prismjs@1.25.0/components/prism-js-extras?no-check";
+//import "https://esm.sh/prismjs@1.25.0/components/prism-js-templates?no-check";
+//import "https://esm.sh/prismjs@1.25.0/components/prism-jsdoc?no-check";
+import "https://esm.sh/prismjs@1.25.0/components/prism-json?no-check";
+import "https://esm.sh/prismjs@1.25.0/components/prism-jsx?no-check";
+//import "https://esm.sh/prismjs@1.25.0/components/prism-markdown?no-check";
+import "https://esm.sh/prismjs@1.25.0/components/prism-rust?no-check";
+//import "https://esm.sh/prismjs@1.25.0/components/prism-toml?no-check";
```

Comment:
why are these commented out?

Reply:
Some of these dont work, so will look into this later